cypress/e2e/models/migration/applicationinventory/assessment.ts

```diff
@@ -192,28 +192,59 @@ export class Assessment extends Application {
         super.edit(updatedValues);
     }
 
+    retake_assessment(
+        risk,
+        stakeholders?: Array<string>,
+        stakeholderGroups?: Array<string>,
+        applicationOpen = false
+    ): void {
+        Application.open();
+        this.selectApplication();
+        clickItemInKebabMenu(this.name, "Assess");
+        cy.wait(SEC);
+        clickByText(button, "Retake");
+        if (stakeholders == undefined && stakeholderGroups == undefined) {
+            expect(
+                false,
+                "At least one arg out of stakeholder or stakeholder groups must be provided !"
+            ).to.equal(true);
+        } else if (stakeholders && stakeholderGroups == undefined)
+            this.perform_assessment(risk, stakeholders, null, (applicationOpen = false));
+        else if (stakeholders == undefined && stakeholderGroups)
+            this.perform_assessment(risk, stakeholderGroups, null, (applicationOpen = false));
+        else if (stakeholders && stakeholderGroups)
+            this.perform_assessment(
```

Comment:
These 3 if else can be combined into one condition .
It will pass null value for whichever parameter doesn't have a value .


 this.perform_assessment(
                risk,
                stakeholder,
                stakeholderGroups,
                (applicationOpen = false)
            );